alerts/openshift-virtualization-operator/KubeVirtHyperconvergedClusterOperatorPodSchedulingMightBeImbalanced.md

```diff
@@ -0,0 +1,21 @@
+# KubeVirtHyperconvergedClusterOperatorPodSchedulingMightBeImbalanced 
+
+## Meaning 
+
+This alert fires when a node contains more than then the maximum configured 
+number of images. The problem happens because the list of images on a node
```

Comment:
I am not sure what this means. How is the problem related to the list of images being shortened to "50"? Are you trying to say that the problem is **diagnosed** when the list of images is shortened to 50?

Reply:
The problem is that the node image list is full, therefore the scheduler will not try to schedule pods that need new images in that node. The issue is described in https://github.com/openshift/openshift-docs/pull/41303/files

Reply:
OK, I suggest this rewording. 

When a node reaches the maximum number of images (default: 50), the scheduler will not schedule pods that require new images on that node.

Reply:
everywhere in openshift we use the initial wording:
https://github.com/openshift/openshift-docs/pull/51515/files
https://github.com/openshift/openshift-docs/pull/51060/files
https://github.com/openshift/openshift-docs/pull/41303/files

Reply:
All I can say is that we should never have used that wording. It makes no sense.

Reply:
updated, wdyt?